docs/development.md

```diff
@@ -380,3 +382,25 @@ volumes:
 ```
 
 Access the service by service name from the `frappe` development container. The above service will be accessible via hostname `postgresql`. If ports are published on to host, access it via `localhost:5432`.
+
+## Using Cypress UI tests
+
+To run cypress based UI tests in a docker environment, follow the below steps:
+
+1. Install and setup X11 tooling on VM using the script `install_x11_deps.sh`
+
+```shell
+  sudo ./install_x11_deps.sh
```

Comment:
sudo bash install_x11_deps.sh

It'll ensure execution even if file doesn't have exec permission